tests/ExampleApps/CoreExampleApp/CoreExampleApp.cpp

```diff
@@ -12,4 +83,30 @@ int main( int /*argc*/, char** /*argv*/ ) {
     auto resourcesPaths = Resources::getBaseDir();
     LOG( Utils::logINFO ) << "Executable is located at : " << resourcesPaths;
     LOG( Utils::logINFO ) << topo.has_vertex_texcoords1D();
+
+    printWedgesInfo( topo );
+
+    LOG( Utils::logINFO ) << "delete face " << *topo.faces_sbegin();
+    topo.delete_face( *topo.faces_begin(), true );
+
+    printWedgesInfo( topo );
+
+    for ( auto itr = topo.halfedges_sbegin(); itr != topo.halfedges_end(); ++itr )
+    {
+        if ( topo.is_collapse_ok( *itr ) )
+        {
+            LOG( Utils::logINFO ) << "collapse he " << *itr << topo.from_vertex_handle( *itr )
+                                  << "->" << topo.to_vertex_handle( *itr );
+            topo.collapse( *itr );
+            break;
+        }
+    }
+
+    printWedgesInfo( topo );
+    LOG( Utils::logINFO ) << "garbage collection ";
+    topo.garbage_collection();
+    printWedgesInfo( topo );
+
+    auto boxOut1 = topo.toTriangleMesh();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'boxOut1' is assigned a value that is never used.](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=4930613)